README.md

```diff
@@ -1,33 +1,48 @@
+
 [![ko-fi](https://img.shields.io/badge/Support%20me%20on-Ko--fi-FF5E5B.svg?logo=ko-fi&style=flat-square)](https://ko-fi.com/mastercoms)
 [![Liberapay](https://img.shields.io/liberapay/receives/mastercoms.svg?logo=liberapay&style=flat-square)](https://liberapay.com/mastercoms/)
 [![Steam donate](https://img.shields.io/badge/Donate%20via-Steam-00adee.svg?style=flat-square&logo=steam)](https://steamcommunity.com/tradeoffer/new/?partner=85845165&token=M9cQHh8N)
 [![Join the Discord chat](https://img.shields.io/badge/Discord-%23comtress--client-7289da.svg?style=flat-square&logo=discord)](https://discord.gg/CuPb2zV)
 
 
-# Team Comtress 2
+# Welcome to Team Comtress 2!
+Before sharing your ideas and feedback, it's important to have a good understanding of what this project is.
+### What *is* TC2?
+Team Comtress 2 is a fork of an older version of Team Fortress 2, with a community developer team fixing bugs, improving performance, and adding quality of life features, with the goal of having those changes pulled upstream by Valve to the modern game.
+
+### What is TC2 *not*?
+* TC2 is *not* a "pro mod"
+
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
+
+* A vision for the "good old days" of TF2
+
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
+
+* The modern game, but with new content/updates
 
-Team Fortress 2, but with a lot of fixes, quality of life improvements and performance optimizations!
+You're looking for [Creators.TF](https://creators.tf/).

```

Comment:
Copy the beginning "TC2 is *not*" used in "TC2 is *not* a "pro mod"" to the other items like " A vision for the "good old days" of TF2" and "The modern game, but with new content/updates" for consistency

Reply:
meant to do this originally and forgot halfway through, lol